File: Python/Ex-52_Teams.py
# 
# Ex-52_Teams.py
# Created by Sebastian Jusca on 17/04/2023.
#
while True:
	try:
		h, mn, s = input("Enter your time in format h.min.s: ").split('.')
		break
	except ValueError:
		print('Enter three numbers with a colon between.')
sc = int(h)*3600 + int(mn)*60 + int(s)

print('{}h {}min {}s are {} seconds in total.'.format(h, mn, s, sc))

File: C/Ex-52_Teams.c
//
//  Ex-52_Teams.c
//  Created by Sebastian Jusca on 29/03/2023.
//

#include <stdio.h>

int main()
{
    int h, min, s, seconds;
    
    printf("Enter number of hours: ");
    scanf("%d",& h);
    
    printf("Enter number of minutes: ");
    scanf("%d",& min);
    
    printf("Enter number of seconds: ");
    scanf("%d",& s);
    
    seconds = (h * 3600) + (min * 60) + s;
    
    printf("%d:%d:%d are in total %d seconds.\n", h, min, s, seconds);
    
    return 0;

}
